src/components/React/Controls.js

```diff
@@ -75,37 +55,29 @@ export default connect(mapStateToProps, {
       this.zSlideHandleChange = this.zSlideHandleChange.bind(this);
       this.volumeSelectChanged = this.volumeSelectChanged.bind(this);
       this.channelSelectChanged = this.channelSelectChanged.bind(this);
-      this.volumeChangedEvent = this.volumeChangedEvent.bind(this);
       this.resetCamera = this.resetCamera.bind(this);
     }
 
     handleCheckBoxInputChange(event) {
```

Comment:
the name of this method vs the body seems a bit discordant.  rename the function? why check if it's a checkbox?